tests/mesh/test-image.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2019 - 2022 Geode-solutions
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include <geode/tests_config.h>
+
+#include <geode/basic/assert.h>
+#include <geode/basic/attribute_manager.h>
+#include <geode/basic/logger.h>
+#include <geode/basic/rgb_color.h>
+
+#include <geode/mesh/core/regular_grid_surface.h>
+#include <geode/mesh/io/regular_grid_input.h>
+#include <geode/mesh/io/regular_grid_output.h>
+
+#include <geode/io/mesh/common.h>
+
+void test_jpg_from_gimp_input()
+{
+    auto grid = geode::load_regular_grid< 2 >(
+        absl::StrCat( geode::data_path, "grid_image_from_gimp.jpg" ) );
+    auto image_attribute =
+        grid->cell_attribute_manager()
+            .find_or_create_attribute< geode::VariableAttribute,
+                geode::RGBColor >( "RGB_data", geode::RGBColor() );
+    OPENGEODE_EXCEPTION( image_attribute->value( 300 ).red() == 102,
```

Comment:
Should we check the loaded attribute values? Maybe by loading a RegularGrid in og_rgd2d containing the answer attribute to compare to?